rental.rb

```diff
@@ -0,0 +1,20 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+  end
+
+  def rent_book=(book)
+    @book = book
+    book.add_rentals(self) unless book.rentals.include?(self)
+  end
+
+  def book_renter=(person)
+    @person = person
+    person.rent_book(self) unless person.books_rented.include?(self)
+  end
```

Comment:
- [ ] in this case you don't create methods for this relationship. The constructor of Rental should be modified so Book and Person are set in it to break the many-to-many relationship, please have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more details